prime-router/metadata/fhir_mapping/hl7/datatype/Annotation.yml

```diff
@@ -13,15 +16,15 @@ time:
   valueOf: "GeneralUtils.dateTimeWithZoneId(dateTimeIn,ZONEID)"
   expressionType: JEXL
   vars:
-    dateTimeIn: NTE.6 | NTE.7
+    dateTimeIn: NTE.6
 
 text:
   type: STRING
   valueOf: $annotationText | NTE.3
   required: true
   expressionType: HL7Spec
 
-extension_1:
+extension_1-noteSource:
```

Comment:
This extension/mapping is not called out in the mapping inventory?

Reply:
It is not, but it is very helpful when debugging